# The original position of turbines and corners of Robin Rigg wind farm 
turbines_x: [46.2900000000000,46.1950000000000,46.1500000000000,46.0560000000000,45.9550000000000,45.8670000000000,45.7450000000000,46.0350000000000,45.9360000000000,45.8400000000000,45.7330000000000,45.6420000000000,45.5370000000000,45.9110000000000,45.8100000000000,45.7020000000000,45.6070000000000,45.5080000000000,45.4150000000000,45.3140000000000,45.7580000000000,45.6710000000000,45.5710000000000,45.4690000000000,45.3750000000000,45.2790000000000,45.1700000000000,45.0750000000000,45.5340000000000,45.4650000000000,45.3410000000000,45.2440000000000,45.1500000000000,45.0480000000000,44.9470000000000,44.8510000000000,45.3070000000000,45.2050000000000,45.1090000000000,45.0190000000000,44.9240000000000,44.8290000000000,44.7330000000000,44.6380000000000,44.9830000000000,44.8730000000000,44.7860000000000,44.6990000000000,44.6040000000000,44.5090000000000,44.4240000000000,44.6200000000000,44.5300000000000,44.4480000000000,44.3620000000000,44.2720000000000,44.1940000000000,44.3000000000000,44.2100000000000,44.1200000000000]
turbines_y: [42.0310000000000,41.6070000000000,42.5600000000000,42.1880000000000,41.7760000000000,41.3940000000000,41.0290000000000,43.1270000000000,42.7140000000000,42.3400000000000,41.9280000000000,41.5580000000000,41.1710000000000,43.7000000000000,43.2970000000000,42.8700000000000,42.4960000000000,42.0930000000000,41.7210000000000,41.3180000000000,44.2070000000000,43.8630000000000,43.4490000000000,43.0410000000000,42.6590000000000,42.2560000000000,41.8700000000000,41.4820000000000,44.4100000000000,43.9950000000000,43.5930000000000,43.1980000000000,42.8160000000000,42.4050000000000,42.0250000000000,41.6320000000000,44.5890000000000,44.1860000000000,43.7500000000000,43.3540000000000,42.9760000000000,42.5690000000000,42.1820000000000,41.7770000000000,44.3330000000000,43.9130000000000,43.5260000000000,43.1300000000000,42.7330000000000,42.3290000000000,41.9250000000000,44.0850000000000,43.6880000000000,43.2920000000000,42.8960000000000,42.5000000000000,42.1280000000000,43.8520000000000,43.4480000000000,43.0600000000000]
corners_x: [46.5600000000000,45.8300000000000,44.6000000000000,43.8800000000000,43.9800000000000,45.4100000000000,45.9500000000000]
corners_y: [41.7300000000000,40.6100000000000,41.3000000000000,42.3500000000000,44.2100000000000,44.9600000000000,44.4600000000000]